archive/d/dart/remove-all-whitespace.dart

```diff
@@ -5,10 +5,8 @@ void main(List<String> argv){
     print(error_message);
     return;
   }
-  String? sentence = argv[0];
-
   //Convert  \r, ]n and \t explicitly into carriage return, newline and tab
-  String raw = sentence.replaceAll(r'\t', '\t').replaceAll(r'\r', '\r').replaceAll(r'\n', '\n');
+  String raw = argv[0].replaceAll(r'\t', '\t').replaceAll(r'\r', '\r').replaceAll(r'\n', '\n');
```

Comment:
What I'm saying is that you done need this line at all. The `.replaceAll` function removes whitespace. If you are trying to add whitespace characters from the command line using `\r`, etc., that will not work. What tests is doing is actually passing those whitespace character.